### DISCUSSION TITLE
Passing information between ViewModels w/ ReactiveUI

### DISCUSSION THREAD

**Discussion by itslyn** [AvaloniaUI/Avalonia#14854] [Q&A] [answered]
I have some child ViewModels & Views, and a parent ViewModel & View.

In my parent View, I have a TransitioningContentControl that displays the current child View:
`<TransitioningContentControl Content="{Binding CurrentPage}"/>`

Here is the code for the parent's ViewModel:
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/f9be061c-8c05-4376-98ac-529d2940cc72)

When the page changes and a new child ViewModel is used as the page, I want the user to be able to input data into some fields on the child page. When the user exits the page by pressing the back button IN the child's view, I want the child ViewModel to pass back all the information back to the parent ViewModel and prompt the parent ViewModel to go back to the default child page.

How do I send properties from my child ViewModel back to the parent ViewModel?

**Comment by thevortexcloud** [comment, marked as answer]
You can use an event, an interaction, or just put the command handler in the parent and read the result from the child. There are many ways to do this.
